.werft/jobs/build/installer/post-process.sh

```diff
@@ -287,7 +287,11 @@ while [ "$documentIndex" -le "$DOCS" ]; do
       yq r k8s.yaml -d "$documentIndex" > /tmp/"$NAME"-"$KIND"-overrides.yaml
       # Parse and update the JSON, and write it to a file
       yq r /tmp/"$NAME"-"$KIND"-overrides.yaml 'data.[config.json]' \
-      | jq ".service.address = $WS_DAEMON_PORT" > /tmp/"$NAME"-"$KIND"-overrides.json
+      | jq ".service.address = $WS_DAEMON_PORT" \
+      | jq ".daemon.cpulimit.enabled = true" \
+      | jq ".daemon.cpulimit.totalBandwidth = 12" \
```

Comment:
I think we have to consider the case of a large VM.

Reply:
I'm afraid of that it affects the integration time. E.g. there is a k3s test case, and this workspace needs a lot of resources to start up k3s.
I am also concerned that these limitations may deplete resources and cause integration-agent to fail to launch. Can you run the test in a normal VM (not large) and see what happens?